installer/build/container/push.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -eu
+
+OVA_REV=$(git rev-parse --verify --short=8 HEAD)
+IMAGE="cluster-api-ova-build"
+# TODO(frapposelli): find a better home for the build container
```

Comment:
👍  agree that a official account is needed

Reply:
moved to `gcr.io/cnx-cluster-api/cluster-api-ova-build`